scripts/dist-git/mkbranch

```diff
@@ -30,6 +30,7 @@ IGNORE=
 BRANCH=""
 PACKAGES=""
 SRC_BRANCH="master"
+AUTHOR="Fedora Release Engineering <rel-eng@lists.fedoraproject.org>"
```

Comment:
Agreed with @xsuchy, more on this -- changing of $AUTHOR per-copr-instance was somewhat challenging before, that's what #9 was mostly about.